Documentation/proxy.md

```diff
@@ -34,4 +34,65 @@ etcd -proxy on -listen-client-urls http://127.0.0.1:8080 -discovery https://disc
 #### Fallback to proxy mode with discovery service
 If you bootstrap a etcd cluster using [discovery service][discovery-service] with more than the expected number of etcd members, the extra etcd processes will fall back to being `readwrite` proxies by default. They will forward the requests to the cluster as described above. For example, if you create a discovery url with `size=5`, and start ten etcd processes using that same discovery url, the result will be a cluster with five etcd members and five proxies. Note that this behaviour can be disabled with the `proxy-fallback` flag.
 
+### Promote a proxy to a member of etcd cluster
+
+A Proxy is not part of the core etcd cluster. A proxy will not promote itself to an etcd member automtically in any case.
+
+If you want to promote a proxy to an etcd member, there are four steps you need to follow:
+
+- add the proxy node as an etcd member into the existing cluster
+- stop the etcd process or service
+- remove the existing proxy data directory
+- restart the etcd process with member configuration
+
+
+#### Example
+
+We assume you have a one member etcd cluster with one proxy. The cluster information is listed below:
+
+|Name|Address|
+|------|---------|
+|infra0|10.0.1.10|
+|proxy0|10.0.1.11|
+
+This example walks you through a case that you promote one proxy to an etcd member. The cluster will become a two member cluster after finishing the four steps.
+
+### Add a new member into the existing cluster
+
```

Comment:
Add a quick sentence above the example:

"First, use `etcdctl` to add the member to the cluster, which will output the environment variables need to correctly configure the new member:"